adapters/mobfoxpb/mobfoxpb.go

```diff
@@ -0,0 +1,131 @@
+package mobfoxpb
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/buger/jsonparser"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+// MobfoxpbAdapter adapter structure
+type MobfoxpbAdapter struct {
+	URI string
+}
+
+// Builder builds a new instance of the Adman adapter for the given bidder with the given config.
```

Comment:
Copy/Paste Mistake: Please change Adman to your bidder name.

Reply:
done